tests/integration/src/test/java/org/apache/pulsar/tests/integration/messaging/PersistentTopicMessagingTest.java

```diff
@@ -63,4 +63,9 @@ public void testNonPartitionedTopicMessagingWithKeyShared(String serviceUrl) thr
     public void testPartitionedTopicMessagingWithKeyShared(String serviceUrl) throws Exception {
         partitionedTopicSendAndReceiveWithKeyShared(serviceUrl, true);
     }
+
+    @Test(dataProvider = "ServiceUrls")
+    public void testResetCursorCompatibility(String serviceUrl) throws Exception {
+        resetCursorCompatibility(serviceUrl);
```

Comment:
It's need to move the test under the package `org.apache.pulsar.tests.integration.backwardscompatibility` Otherwise the the test is using the latest broker and latest client, so it can't check the compatibility.